model/optimality_objective.py

```diff
@@ -65,9 +65,11 @@ def add_objective_for_optimal_solution(self, weights, f, g, delta):
         self.model.setObjective(
             quicksum(f["plus"][e] - f["minus"][e] for e in self.employees)
             + weights["lowest fairness score"] * (g["plus"] - g["minus"])
-            - weights["demand_deviation"]
+            - weights["demand deviation"]
```

Comment:
Immediately I do not see the need for three different weights for demand deviation. I would think just one weight for positive deviation might suffice if the negative deviation is used as a baseline. What is your intention on using three weights?

Reply:
Good point! The standard "demand deviation" could be removed completely! I still want to keep the "excess demand deviation factor" and "deficit demand deviation factor", as i might be desirable to down or up-scale both of these weights from the reference value 1.

Reply:
Yeah, you have to at least have two weights here. If not it would be impossible to scale according to `time_step` and `num_employees`